tools/provision

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+BASEDIR=`dirname $0`/..
+
+if [ ! -d "$BASEDIR/zulip-api-py2-venv" ]; then
+    virtualenv $BASEDIR/zulip-api-py2-venv
+    echo "Virtualenv created."
+fi
+
+VENVBINDIR="bin"
+if [[ "$OSTYPE" == "cygwin" ]]; then
```

Comment:
@timabbott since this works for all Linux users anyway (and for me with `msys`, just tested it), I think this is ready to merge. Checking the ostype variable here is not optimal, since it doesn't even exist it cmd :). I'll refactor this to rather check for the existence of the directory.